deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -840,7 +840,7 @@ class ChatViewController: UITableViewController {
     private func updateTitle(chat: DcChat) {
         let titleView =  ChatTitleView()
 
-        var subtitle = "ErrSubtitle"
+        var subtitle = ""
```

Comment:
this `ErrSubtitle` was visible before - as the "just deleted ChatViewController" is skipped on going back now, `ErrSubtitle` should no longer be shown.

however, there is also not much in it, the chat title is blank as well - and if the chat flashes up shortly for whatever reason, there is no need to show a weird string.